domains/certificates/RegistryConnector/Worker/Program.cs

```diff
@@ -24,6 +24,7 @@
 var log = new LoggerConfiguration()
     .Filter.ByExcluding("RequestPath like '/health%'")
     .Filter.ByExcluding("RequestPath like '/metrics%'")
+    .MinimumLevel.Override("Microsoft.EntityFrameworkCore.Database.Command", Serilog.Events.LogEventLevel.Warning)
```

Comment:
Doesn't this mute all Information log entries and not just the ones spamming?

Reply:
No